Mathlib/CategoryTheory/Elements.lean

```diff
@@ -64,6 +64,16 @@ instance categoryOfElements (F : C ⥤ Type w) : Category.{v} F.Elements where
   id p := ⟨𝟙 p.1, by aesop_cat⟩
   comp {X Y Z} f g := ⟨f.val ≫ g.val, by simp [f.2, g.2]⟩
 
+/-- The functor mapping a functor (C ⥤ Type w) to its category of elements -/
+def Functor.ElementsFunctor :  (C ⥤ Type w) ⥤ Cat where
```

Comment:
I would suggest introducing an auxiliary definition, which may be useful otherwise:
```lean
@[simps]
def NatTrans.mapElements {F G : C ⥤ Type w} (φ : F ⟶ G) : F.Elements ⥤ G.Elements where
  obj := fun ⟨X, x⟩ ↦ ⟨X, φ.app X x⟩
  map {X Y} f := CategoryOfElements.homMk _ _ f.1 (by
    sorry)
```
Then, `ElementsFunctor` should be renamed `elementsFunctor`. (`Elements` is capitalized because it is a type, but `elementsFunctor` is a functor.)